### PR TITLE
WT-7599 Compatibility test - create multiple config files for each release

### PR DESCRIPTION
**Pull request by raviprakashgiri29**
Updated the compatibility script to create multiple config files each per release, so that each branch is provided a CONFIG file that it understands.

### CODE REVIEW

File: test/evergreen/compatibility_test_for_releases.sh
Comment (raviprakashgiri29) [inline test/evergreen/compatibility_test_for_releases.sh:225]: Removed "reopen" option because I am passing the config file to the test-format that it understands and -c is not compatible with -R.

Comment (keithbostic) [reply test/evergreen/compatibility_test_for_releases.sh:225]: I'm confused, won't this mean we remove the existing database and re-create a new one? Doesn't that mean we've removed the point of this test, that is, to run a single database under multiple releases?

Comment (raviprakashgiri29) [reply test/evergreen/compatibility_test_for_releases.sh:225]: The main reason for removing the "-R" option (reopen the existing database) because I want to pass the CONFIG to the test-format binary that it understands. For eg - While doing an upgrade/downgrade test 4.2 test-format binary is run on a 4.4 CONFIG which 4.2 does not understand(it has some configurations which is not supported by older release).
@keithbostic  Please suggest if there is any other better option to handle these scenarios.

Comment (keithbostic) [reply test/evergreen/compatibility_test_for_releases.sh:225]: I have not looked at this, but from my recollections, I suspect that `-R` has to co-exist with `-c`.

Comment (raviprakashgiri29) [reply test/evergreen/compatibility_test_for_releases.sh:225]: @keithbostic I created a ticket WT-7721 to update test-format binary to reopen an existing database with different CONFIG, to `-c` and `-R` work together. I have also requested a backport of these changes to the 5.0, 4.4, and 4.2 branches. 

I will keep this ticket WT-7599 on hold until I merge WT-7721.

File: test/evergreen/compatibility_test_for_releases.sh
Comment (keithbostic) [inline test/evergreen/compatibility_test_for_releases.sh:135]: extra blank line.

**Review by keithbostic — APPROVED** [review]
lgtm

**Comment by raviprakashgiri29** [comment]
@tammybailey  @AlexeyAnisimovAU Can you please review the changes?

**Review by AlexeyAnisimovAU — APPROVED** [review]
LGTM

**Review by tammybailey — COMMENTED** [review]
lgtm

**Review by tammybailey — APPROVED** [review]

**Pull request merged**
